File: lib/models/Utente.java
package models;

/**
 * 
 * @author Allari Edoardo
 * @version 1.0.0 - 14/03/2020
 * 
 */

public class Utente{


    private long Id;

    private String Username;

    private String Password;

    private String Email;

    private String SALT;

    private String Roles;

    public Utente() {
    }

    public Utente(long id, String username, String password, String email, String sALT, String roles) {
        this.Id = id;
        this.Username = username;
        this.Password = password;
        this.Email = email;
        this.SALT = sALT;
        this.Roles = roles;
    }

    public long getId() {
        return Id;
    }

    public void setId(long id) {
        Id = id;
    }

    public String getUsername() {
        return Username;
    }

    public void setUsername(String username) {
        Username = username;
    }

    public String getPassword() {
        return Password;
    }

    public void setPassword(String password) {
        Password = password;
    }

    public String getEmail() {
        return Email;
    }

    public void setEmail(String email) {
        Email = email;
    }

    public String getSALT() {
        return SALT;
    }

    public void setSALT(String SALT) {
        this.SALT = SALT;
    }

    public String getRoles() {
        return Roles;
    }

    public void setRoles(String roles) {
        Roles = roles;
    }
}

File: lib/model/Utente.dart

class Utente {

  int _id;
  String _username;
  String _password;
  String _email;
  String _salt;
  String _roles;


  Utente();

  String get roles => _roles;

  set roles(String value) {
    _roles = value;
  }

  String get salt => _salt;

  set salt(String value) {
    _salt = value;
  }

  String get email => _email;

  set email(String value) {
    _email = value;
  }

  String get password => _password;

  set password(String value) {
    _password = value;
  }

  String get username => _username;

  set username(String value) {
    _username = value;
  }

  int get id => _id;

  set id(int value) {
    _id = value;
  }




}